### ISSUE TITLE
Endless Loading on launch

### ISSUE THREAD

**Issue by irudoy** [avocode/avocode#326]
I launched Avocode, and got endless Loading

-----

```
TypeError: object is not a function
    at t.<anonymous> (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/@avocode/core/lib/avocode.js:1:26467)
    at t.EventEmitter.emit (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/@avocode/core/node_modules/eventemitter2/lib/eventemitter2.js:339:22)
    at e._onUserData (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/@avocode/core/lib/user.js:1:8253)
    at /Applications/Avocode.app/Contents/Resources/app.asar/node_modules/@avocode/core/lib/user.js:1:342
    at Request._callback (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/@avocode/core/lib/api.js:1:2348)
    at Request.self.callback (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/@avocode/core/node_modules/request/request.js:197:22)
    at emitTwo (events.js:92:20)
    at Request.emit (events.js:169:7)
    at Request.<anonymous> (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/@avocode/core/node_modules/request/request.js:1050:14)
    at emitOne (events.js:82:20)
    at Request.emit (events.js:166:7)
    at IncomingMessage.<anonymous> (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/@avocode/core/node_modules/request/request.js:996:12)
    at emitNone (events.js:72:20)
    at IncomingMessage.emit (events.js:163:7)
    at _stream_readable.js:891:16
    at process._tickCallback (node.js:357:13)
```

**Comment by rychlis** [comment]
Could you send us a screenshot please? Thanks a lot. Does restarting help or is the error permanent?

**Comment by irudoy** [comment]
Screenshot: https://www.dropbox.com/s/6iz4o61monsjw3h/Screen%20Shot%202015-07-02%20at%2010.14.29.png?dl=0
The error is permanent.

**Comment by rychlis** [comment]
This should be fixed in 2.3.0
